lc0/LC0VSProj/LC0VSProj.vcxproj

```diff
@@ -83,7 +87,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <WholeProgramOptimization>true</WholeProgramOptimization>
     <CharacterSet>MultiByte</CharacterSet>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v140</PlatformToolset>
```

Comment:
I'm not sure what's going on here, my MSVS (2015) is unable to open that project anyway.

Reply:
That's weird as v140 is the 2015 toolchain - perhaps it's the Windows SDK version?